src/coreclr/src/jit/importer.cpp

```diff
@@ -3506,6 +3506,11 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
 
                 return hwintrinsic;
             }
+
+            if ((ni > NI_SIMD_AS_HWINTRINSIC_START) && (ni < NI_SIMD_AS_HWINTRINSIC_END))
+            {
+                return impSimdAsHWIntrinsic(ni, clsHnd, method, sig, mustExpand);
```

Comment:
If the intrinsic isn't currently handled or ends up returning `nullptr`, we will currently still hit the existing `impSIMDIntrinsic` path and produce a GT_SIMD node later.